packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -39,24 +34,18 @@ const navigationNodeToEuiItem = (
     [`nav-item-isActive`]: isSelected,
   });
 
-  const getRenderItem = (): RenderItem | undefined => {
-    if (!isExternal || item.renderItem) {
-      return item.renderItem;
-    }
-
-    return () => (
-      <div className="euiSideNavItemButton" data-test-subj={dataTestSubj}>
-        <EuiLink href={href} external color="text">
-          {item.title}
-        </EuiLink>
-      </div>
-    );
-  };
+  // DEBUG isActive state updates
+  // console.log('isActive?', id, item.isActive);
 
   return {
     id,
-    name: item.title,
+    isGroupTitle: item.isGroupTitle,
+    title: item.title,
     isSelected,
+    accordionProps: {
+      initialIsOpen: true, // FIXME open state is controlled on component mount
```

Comment:
In this PR, all sub-accordions in a nav tree will be expanded by default: `initialIsOpen` controls whether the accordion items of EuiCollapsibleNavItem are open by default on component mount.

More work is needed after this PR to have the Kibana component to take control of open/closed state of accordion nav items with the "active nodes" state we track in this component.

With "active nodes" state changes, accordion items the user explicitly opened need to remain open, even if none of the items are selected - that means extra state we need to track in this component.

cc @cee-chen @sebelga

Reply:
Ok. I'll have a look into it in a future PR. For now keeping them open works 👍

Reply:
Filed https://github.com/elastic/kibana/issues/167328